src/index.html

```diff
@@ -1,12 +1,414 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__content">
+    <header class="header">
+      <img src="./images/logo.png" class="logo" alt="logo" />
+      <nav class="nav">
+        <div class="list">
+          <a href="#" class="list--item is-active">Apple</a>
+          <a href="#" class="list--item">Samsung</a>
+          <a href="#" class="list--item">Смартфоны и телефоны</a>
+          <a
+            href="#"
+            class="list--item"
+            data-qa="nav-hover"
+          >Ноутбуки и компьютеры</a>
```

Comment:
```suggestion
          <a
            href="#"
            class="list__item"
            data-qa="nav-hover"
          >
            Ноутбуки и компьютеры
          </a>
```
item это не модификатор, а элемент, если ты пишешь по БЭМу